src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -153,6 +153,14 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
                             return AuthenticateResult.Skip();
                         }
 
+                        if (Options.SaveToken)
+                        {
+                            ticket.Properties.StoreTokens(new[]
+                            {
+                                new AuthenticationToken { Name = "Bearer", Value = token }
```

Comment:
I don't have enough context to say, I would say we should name it 'correctly' in the context of JwtBearer as opposed to consistently.  I think it's better for it to be as clear as possible what this token is based on the name.  That said, if access_token is correct-ish, consistency is great :)